pip\_services3\_commons.errors package
======================================

Submodules
----------

.. toctree::
   :maxdepth: 4

   pip_services3_commons.errors.ApplicationException
   pip_services3_commons.errors.ApplicationExceptionFactory
   pip_services3_commons.errors.BadRequestException
   pip_services3_commons.errors.ConfigException
   pip_services3_commons.errors.ConflictException
   pip_services3_commons.errors.ConnectionException
   pip_services3_commons.errors.ErrorCategory
   pip_services3_commons.errors.ErrorDescription
   pip_services3_commons.errors.ErrorDescriptionFactory
   pip_services3_commons.errors.FileException
   pip_services3_commons.errors.InternalException
   pip_services3_commons.errors.InvalidStateException
   pip_services3_commons.errors.InvocationException
   pip_services3_commons.errors.NotFoundException
   pip_services3_commons.errors.UnauthorizedException
   pip_services3_commons.errors.UnknownException
   pip_services3_commons.errors.UnsupportedException

Module contents
---------------

.. automodule:: pip_services3_commons.errors
   :members:
   :undoc-members:
   :show-inheritance:
